ci/unreal-engine.version

```diff
@@ -1 +1 @@
-HEAD 4.22-SpatialOSUnrealGDK
\ No newline at end of file
+HEAD feature/bk-provide-starter-content
```

Comment:
We probably want them both to be merged in at the same time, so we should change this back to the head of 4.22.

Reply:
Yes, as far as I understand - the current workflow is to first get the engine change merged, and then I will update this file.